src/services/Api.tsx

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+// import { useState, useEffect } from 'react';
+// import { IWeather } from '../interfaces/IWeather';
+
+export const weatherData = axios.create({
+  baseURL: 'http://mock-api-call/weather/get-weather',
```

Comment:
just a nit pick but you can access the base api from the .env file by calling `process.env.REACT_APP_API_URL`